app/bundles/ReportBundle/Views/Report/form.html.php

```diff
@@ -136,7 +141,54 @@
                                 </div>
                             </div>
                         </div>
-
+                        <div class="tab-pane fade bdr-w-0" id="schedule-container">
+                            <div class="row">
+                                <div class="col-md-12">
+                                    <div class="pa-md">
+                                        <div class="checkbox">
```

Comment:
Remove this DIV as class=checkbox causes the yes/no buttons to be a little wonky

Reply:
Fixed